src/test/java/com/thirdcc/webapp/web/rest/UserResourceIT.java

```diff
@@ -710,6 +720,78 @@ public void getAllNotEventCrewUsers_WithEventCrewUser() throws Exception  {
             .andExpect(jsonPath("$.[*].imageUrl").value(not(hasItem(DEFAULT_IMAGEURL))));
     }
 
+    @Test
+    @Transactional
+    public void getUserWithoutFamily_ShouldReturn200() throws Exception {
+        userRepository.deleteAll();
```

Comment:
No need delete again since you have did in @AfterEach right?

Reply:
Resolved in 5940d2bd46afc1053fdcb4d95e313b57bdf44cde